### DISCUSSION TITLE
How to merge changes in 2 forks into my clone of the original  project?

### DISCUSSION THREAD

**Discussion by fdavis99** [community/community#29080] [New to GitHub] [answered]
I cloned blackducksoftware/json-splitter to my machine. The project has 2 forks (pjsnps/json-splitter and petergreen86/json-splitter) each of which has useful changes. Neither has been merged back to the original project.

How can I merge the changes from each fork into my clone of the original? 

I ran "git remote add json-splitter git@github.com:pjsnps/json-splitter.git" per https://stackoverflow.com/a/1123364/1701482, but I don't know what to do next, nor how to get the changes from the petergreen86 fork.

**Comment by airtower-luna** [comment, marked as answer]
Check [Git Basics - Working with Remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) for a comprehensive explanation.

In (very) short, once you've added a remote you can *fetch* from there (to get actual branches, commits, etc.) with `git fetch <remote-name>`. And then the branches are available to you namespaced with the remote name (e.g. `json-splitter/main` or something for the remote you've already added). You can checkout, merge, or whatever. :slightly_smiling_face:

**Reply by fdavis99** [reply]
Found a solution: https://stackoverflow.com/questions/1123344/merging-between-forks-in-github which worked. Thanks, @airtower-luna
